icons/primeng/primeng-plain.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#000" d="M64 0 4.182 21.248l9.123 78.783L64 128l50.695-27.969 9.123-78.783zM44.553 22.463h8.927l-4.464 10.2-13.389-1.276zm10.84 0h7.013v36.332H58.58l-27.416-10.2-3.826-15.933 22.974 1.861 5.08-12.06zm9.564 0h7.014l5.08 12.062 23.611-1.863-3.826 15.934L69.42 58.795h-4.463zm8.926 0h8.926l8.927 8.924-13.39 1.275-4.463-10.2zM31.8 51.146l7.012 2.55 7.013 7.011 5.738-1.275-4.46 6.373v19.76L31.8 72.815v-21.67zm63.76 0-.002 21.67-15.3 12.748v-19.76l-4.464-6.372 5.736 1.275 7.016-7.012zm-41.524 8.192h.002l3.182 1.916h12.71l3.18-1.916 5.086 7.664v28.74l-3.814 5.748-4.45 4.47H57.218l-4.45-4.47-3.816-5.748v-28.74l5.086-7.664zM38.814 81.102l8.29 7.011v9.56l-8.29-8.284zm49.733 0v8.287l-8.287 8.285v-9.56z"/></svg>
```

Comment:
The `fill="#000"` can be removed, since this is the default, but this is not a blocker IMHO.